README.md

```diff
@@ -245,6 +250,26 @@ This provides you the power of entire Javascript to build your desired HTML stri
 | maxLabel | String | "" | Text for the higher end of the range slider |
 | prefix | String | "$" | Prefix text to be added to the range widget value. Example "$" for price facet |
 
+## Selected Facets Config
```

Comment:
please add this in the the facet config list above and link it to this section like we have done for other nested configs

Reply:
This is still not added in the facets section above right?